modules/zookeeper/pom.xml

```diff
@@ -45,44 +45,7 @@
             <groupId>org.apache.curator</groupId>
```

Comment:
We don't need this at all
All we need is `curator-test`

It is defined few lines after.

Reply:
`curator-framework` need for ZookeeperDiscoverySpiTestBase